README.md

```diff
@@ -101,6 +101,8 @@ The server will automatically restart anytime you save a `.ts` or `.js` file wit
 
 You can run any command within the container by prefixing it with `docker-compose exec app`, e.g. `docker-compose exec app npm install express`
 
+Next, [seed the DB](#seeding-the-db).
```

Comment:
Was this step also intentionally omitted from the "Natively running" section?

Reply:
Yeah, the seeding section is immediately after that, so I figured people would naturally go to that after `npm run dev`.  Do you think it would be better to repeat it?